dasbus/server/interface.py

```diff
@@ -407,7 +408,7 @@ def _iterate_parameters(cls, member):
             # Check if the type is defined.
             if name not in type_hints:
                 raise DBusSpecificationError(
-                    "Parameter {} doesn't have defined type.".format(name)
+                    "Missing type of parameter '{}'.".format(name)
```

Comment:
I am not entirely sure, but how about of -> for ?

Reply:
Fixed.